examples/GeoData_on_hover.ipynb

```diff
@@ -0,0 +1,124 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
```

Comment:
Do you mind removing the Notebook outputs?

Reply:
Sure, it's clean now!